README.md

```diff
@@ -52,5 +64,81 @@ cd ./ensembl-genomio
 ./scripts/setup/docs/build_sphinx_docs.sh
 ```
 
+## Pipelines
+
+### Initialising and running eHive-based pipelines
+
+Pipelines are derived from [`Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf`](https://github.com/Ensembl/ensembl-hive/blob/version/2.6/modules/Bio/EnsEMBL/Hive/PipeConfig/HiveGeneric_conf.pm),
+or from [`'Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf`](https://github.com/Ensembl/ensembl-hive/blob/version/2.6/modules/Bio/EnsEMBL/Hive/PipeConfig/EnsemblGeneric_conf.pm),
```

Comment:
Single quote on this line without closing quote ( 'Bio...)

Reply:
thanks!